pkg/envoy/secretsync.go

```diff
@@ -35,6 +35,10 @@ const (
 	// easy rotation of keys by, for example, putting the new key first, and the previous key second.
 	// Additional keys are defined by using the Secret key 'tls.sessionticket.key.[1-9]'
 	tlsSessionTicketKeyAttribute = "tls.sessionticket.key"
+
+	// Key for a generic Envoy secret is fixed as 'generic.secret' even though is not as "standard"
+	// as 'tls.crt' and 'tls.key' are via k8s tls secret type.
+	genericSecretAttribute = "generic.secret"
```

Comment:
Maybe just `generic`? It is in a secret already... Or then `envoy.generic`?

Reply:
makes sense - changed to `generic`